### DISCUSSION TITLE
Insert, remove and reorder intents do not work with button type = button

### DISCUSSION THREAD

**Discussion by hauptrolle** [edmundhung/conform#482] [Q&A] [answered]
### Describe the bug and the expected behavior

When adding `type="button"` to an insert, remove or reorder intent the intent is not working anymore.

### Conform version

1.0.1

### Steps to Reproduce the Bug or Issue

1. Checkout example from website: https://conform.guide/intent-button
2. Add `type="button"` to intent buttons
3. It does nothing

**Comment by edmundhung** [comment, marked as answer]
All form intent are designed to work before JS. Making it type button means it will not work the way it is designed. Why do you wanna restrict it?

Anyway, if you really have to do this, you can always setup a onClick handler and trigger the form intent directly:

```jsx
<button type="button" onClick={() => form.reset()} />
```

**Reply by edmundhung** [reply]
Maybe I should make it explicit in the docs that the button is type `submit` even though it is the default type 😓 I can imagine people being used to `type="button"` for a long time and find it confusing.

**Reply by hauptrolle** [reply]
Ahhh I see. I will just use the onClick then. 

background: I am using the new HeadlessUI release where the buttons type defaults to type=button. That's why I was wondering 😀

But a note in the docs would be nice 👍
